### PR TITLE
feat: have action accept more inputs and extrapolate output filename

### PR DESCRIPTION
**Pull request by joshua-stone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ This action expects the following inputs:
 - `image-name`: The name of the ISO to generate.  This does not include the ISO extension.
 - `installer-major-version`: The major version of the Fedora installer to use.  This is usually the same as the image major version.
 - `installer-repo`: Either `development` to grab the latest Fedora installer, or `release` for the latest stable release
-
+- `cpu-arch`: CPU architecture for the installer ISO. Optional, defaults to `x86_64`
```
Comment (akdev1l) [inline README.md:54]: Should this be a list? So we can do 

```
cpu_arch:
  - x86_64
  - aarch64
```

Comment (akdev1l) [reply README.md:54]: This probably  plays well as is with my other pr on main tho

**Review by akdev1l — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -15,10 +15,30 @@ inputs:
...
 
 runs:
   using: composite
   steps:
+    - name: Calculate ISO name
+      id: iso-name
+      shell: bash
+      run: |
+        if [[ -z "${{inputs.output-filename }}" ]]; then
+          ISO_NAME="${{ inputs.image-name }}-${{ inputs.installer-major-version }}-${{ inputs.cpu-arch }}-$(date +%Y%m%d).iso"
```
Comment (p5) [inline action.yml:35]: I think there should be separate inputs for the `installer-major-version` and `fedora-major-version`.  
`installer-major-version` can default to 38, but we may soon want to release 37, 39 or rawhide ISOs, which wouldn't work with this naming pattern - the installer is not available for 37 for example

Comment (p5) [reply action.yml:35]: Or will they always be the same?

Comment (p5) [reply action.yml:35]: Actually, since we're not releasing separate ISOs for each Fedora release, this is good.

**Review by p5 — APPROVED** [review]

**Pull request merged**